src/display_servers/xlib_display_server/xwrap.rs

```diff
@@ -1444,11 +1443,11 @@ impl XWrap {
         }
     }
 
-    pub fn load_colors(&mut self, theme: &ThemeSetting) {
+    pub fn load_colors(&mut self, config: &impl Config) {
         self.colors = Colors {
-            normal: self.get_color(&theme.default_border_color),
-            floating: self.get_color(&theme.floating_border_color),
-            active: self.get_color(&theme.focused_border_color),
+            normal: self.get_color(config.default_border_color()),
```

Comment:
Just for my understanding, why is no borrow needed here?

Reply:
The functions already return a borrowed value.... Though I'm not entirely sure if the Config trait's functions should return owned values or borrowed value :thinking: Right now it's a bit mixed up... There should be a right answer, maybe it will reveal itself in a future PR. Don't hesitate to let me know if you have any idea on this.

Reply:
Oh, thanks should have followed the rabbit hole a bit deeper I guess.
At this point I'm also not sure what would be best.